modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -39,7 +39,7 @@ Note: We have used the `-y` flag to take the initialization defaults. To change
 
 [source,json,indent=0]
 ----
-include::../examples/getting-started/package.json[tag=start-using-package-json]
+include::../examples/getting-started/package.json[tags=**]
```

Comment:
Just a question for my benefit, what does `[tags=**]` do? 
I vaguely remember you mentioned Antora tags in the `package.json`, were they causing issues or just seemed odd?

Reply:
it shows everything, even if not tagged (as .json doesn't have comments so can't be tagged and remain valid)